metadata/src/from_into/v14.rs

```diff
@@ -368,7 +372,10 @@ fn generate_outer_error_enum_type(
                 return None;
             };
 
-            let path = format!("{}Error", pallet.name);
+            // Note:  using the `format!` macro like in `let path = format!("{}Error", pallet.name);`
+            // leads to linker errors about extern function `_Unwind_Resume` not being defined.
```

Comment:
Strange; this isn't expected! Not a big deal but the `alloc::format!` macro should work fine here so I'd be keen to understand this :)

Reply:
So I tried running this no-std-tests on my M1 mac with `cargo +nightly run` and it blew up hideously for me (error below) even without the format thing!

A way I've tested no-std before it to try compiling for a certain architecture that doesn't support std. It's a bit less good in that it doesn't then also test any code, but worked pretty well.

I wonder what needs changing to make this example compile on my mac! I'm a little scared by un-understandable errors :D

(my error:)

```
error: linking with `cc` failed: exit status: 1
  |
  = note: env -u IPHONEOS_DEPLOYMENT_TARGET -u TVOS_DEPLOYMENT_TARGET LC_ALL="C" PATH="/Users/james/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/bin:/opt/homebrew/opt/llvm/bin:/Users/james/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/james/.cargo/bin" VSLANG="1033" ZERO_AR_DATE="1" "cc" "-arch" "arm64" "/var/folders/w9/qpj02kr93657cz711ppx0t000000gn/T/rustc87T3UO/symbols.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.15d5c1azm5qs75er.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.18o25cv5mh1czzjv.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.1bgxi20o8beedxh9.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.1m7v5cg51x1p7t79.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.1mdjt5lke7rk419d.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.1v8ak6ohszt9il8x.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.1v9sz25zdztrk2hg.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.20lu86g86pae7v1.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.269mpyibt1n9z2i8.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.26gmy4iee7clpnjg.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.2827u6krwnwjtvmo.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.28terp055wptt9us.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.2gjo3dujmmxwsynl.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.2hdy6vweutivgntb.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.2ij89yj2b9cfmh0k.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.2kxijmagjgn5c651.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.2l9spa5avf7kavpj.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.2mp073awr9qou2fb.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.2xjmwo260sodaqiq.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.31qn6lw1n4lpjdve.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.347ir8fl8lxog3w6.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.34ay7fxhwl65bj7v.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3g07tcecbxwnnq15.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3ggonenpgi5q0rf7.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3hy4l7kks4k4namn.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3mopnmoc36199hoe.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3mwmtcsm84pap21c.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3pxzeifl0mb14zss.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3qm0plxewj6nsfhf.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3qw0ledh0ykcy9u6.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3sx8ru8t9n82j95.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3t69n7kzsmq96tw3.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.3wrm7uvmrqxraehz.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.400crr6k9d61fu17.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.40vv1ehx1u9exf8h.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.419xohj6in5i57zm.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.49kcb06tsafvm0db.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.49o2xnetxigqdpro.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.4h4ig9h9alzumjiu.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.4l56t28e7lpv17zl.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.4l6ylrtnyepjcwz3.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.4lvkbyvrpemqogh6.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.4w4f3riwmsudy1je.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.4xkclc8ecezl0k83.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.51jptb7x34zdfqf2.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.52x4vlfiymy8pwzd.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.55rg619tv9lf7277.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.5cbdvd0r51grlge5.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.5cfwus2ubakw9ax4.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.7gevje2ejveviaa.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.caeimgud3je6u6x.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.cg62d0ulxjdhpt3.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.er7b56c56du2olw.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.f5h5r5p91v2k9ni.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.fe25tv2z76lodxh.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.gt1xdawrq4l7qvw.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.jv85tnelye4bgri.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.xo8z24wgj0cg9av.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.zq1xrzayduo33qg.rcgu.o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b.1t8ag4zloyspfmmk.rcgu.o" "-L" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps" "-L" "/Users/james/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libsubxt_metadata-8e248289f930cb3c.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libsp_core_hashing-f0c25e154b5c4277.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libsha2-cb2ce43e4b8e194a.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libsha3-5838d6073efb5c3e.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libkeccak-d2d65997e5888b81.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libtwox_hash-857443a2d443ea8b.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libstatic_assertions-9023d874dd538a80.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libblake2b_simd-9fb389da054224a0.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libconstant_time_eq-29dc5e34c80a75b2.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libarrayref-9b1fb0b52a86eb41.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libdigest-4c52f81b6eec5fb1.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libblock_buffer-ec353a40d52c9037.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libcrypto_common-57844e855bcb6cd4.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libgeneric_array-b1f0fee6a20334f9.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libtypenum-775d0f072d4f4abd.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libbyteorder-0820adf1e7c48840.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libhashbrown-5244040c47fd6a4a.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libahash-a7a7d37f197f75c4.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libonce_cell-224b04cb628f70ea.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libzerocopy-c44aef1ea18b0c6c.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/liballocator_api2-ca7226a82670e6c8.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libframe_metadata-e1923b9d96c4e0ee.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libscale_info-799db87831817bb0.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libcfg_if-a70192628636786c.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/liblibc-8cc6a15d6487055d.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libparity_scale_codec-39751d065c7b75ba.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libarrayvec-dff866a194c19e5e.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/libbyte_slice_cast-8fd10abbb6202346.rlib" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/liblibc_alloc-8e6c22beab38a9a5.rlib" "/Users/james/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/liballoc-fce2a9bd028a2944.rlib" "/Users/james/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_core-e5afe4e689972eee.rlib" "/Users/james/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcore-113e29613e2cc3c4.rlib" "/Users/james/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcompiler_builtins-0faa98e433ca4c9a.rlib" "-liconv" "-L" "/Users/james/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib" "-o" "/Users/james/Work/subxt/testing/no-std-tests/target/debug/deps/subxt_core_no_std_tests-c9e698994923ea6b" "-Wl,-dead_strip" "-nodefaultlibs"
  = note: Undefined symbols for architecture arm64:
            "_memcmp", referenced from:
                _$LT$u8$u20$as$u20$core..slice..cmp..SliceOrd$GT$::compare::h2bc851d9c6b79f93 in libscale_info-799db87831817bb0.rlib(scale_info-799db87831817bb0.scale_info.bfaba07d6ed144ac-cgu.08.rcgu.o)
                _$LT$$u5b$A$u5d$$u20$as$u20$core..slice..cmp..SlicePartialEq$LT$B$GT$$GT$::equal::hb5134437d2fec988 in libscale_info-799db87831817bb0.rlib(scale_info-799db87831817bb0.scale_info.bfaba07d6ed144ac-cgu.08.rcgu.o)
            "_memmove", referenced from:
                core::slice::sort::merge_sort::RunVec$LT$RunAllocF$C$RunDeallocF$GT$::remove::he4ebdd000273eb8e in subxt_core_no_std_tests-c9e698994923ea6b.52x4vlfiymy8pwzd.rcgu.o
                alloc::collections::btree::node::BalancingContext$LT$K$C$V$GT$::bulk_steal_left::h7a76fcbed2e55b36 in libframe_metadata-e1923b9d96c4e0ee.rlib(frame_metadata-e1923b9d96c4e0ee.frame_metadata.41e071e3dfd6810e-cgu.03.rcgu.o)
                hashbrown::raw::RawTableInner::prepare_rehash_in_place::h1c0f596ae44a7f5b in libsubxt_metadata-8e248289f930cb3c.rlib(subxt_metadata-8e248289f930cb3c.2vd0twcydxhja38y.rcgu.o)
            "_memset", referenced from:
                _$LT$i8$u20$as$u20$alloc..vec..spec_from_elem..SpecFromElem$GT$::from_elem::ha4d40836b07f28c1 in subxt_core_no_std_tests-c9e698994923ea6b.5cbdvd0r51grlge5.rcgu.o
                _$LT$u8$u20$as$u20$alloc..vec..spec_from_elem..SpecFromElem$GT$::from_elem::he5fc054bff2de0b8 in subxt_core_no_std_tests-c9e698994923ea6b.5cbdvd0r51grlge5.rcgu.o
                hashbrown::raw::RawTableInner::fallible_with_capacity::had20f01794ceec46 in libsubxt_metadata-8e248289f930cb3c.rlib(subxt_metadata-8e248289f930cb3c.4di09es9j9z0uye5.rcgu.o)
                _$LT$alloc..vec..into_iter..IntoIter$LT$T$C$A$GT$$u20$as$u20$core..iter..traits..iterator..Iterator$GT$::next::h984782dda0343e33 in libframe_metadata-e1923b9d96c4e0ee.rlib(frame_metadata-e1923b9d96c4e0ee.frame_metadata.41e071e3dfd6810e-cgu.01.rcgu.o)
                _$LT$alloc..vec..into_iter..IntoIter$LT$T$C$A$GT$$u20$as$u20$core..iter..traits..iterator..Iterator$GT$::next::h148c0c7b754fe643 in libsubxt_metadata-8e248289f930cb3c.rlib(subxt_metadata-8e248289f930cb3c.4f0vy5igmgli8fsf.rcgu.o)
                _$LT$alloc..vec..into_iter..IntoIter$LT$T$C$A$GT$$u20$as$u20$core..iter..traits..iterator..Iterator$GT$::next::h6b190362ceded8d4 in libsubxt_metadata-8e248289f930cb3c.rlib(subxt_metadata-8e248289f930cb3c.4f0vy5igmgli8fsf.rcgu.o)
                _$LT$alloc..vec..into_iter..IntoIter$LT$T$C$A$GT$$u20$as$u20$core..iter..traits..iterator..Iterator$GT$::next::h6e1c6f16b2e57101 in libsubxt_metadata-8e248289f930cb3c.rlib(subxt_metadata-8e248289f930cb3c.4f0vy5igmgli8fsf.rcgu.o)
                ...
            "_memalign", referenced from:
                _$LT$libc_alloc..LibcAlloc$u20$as$u20$core..alloc..global..GlobalAlloc$GT$::alloc::h6ceccb6ff37b97f1 in subxt_core_no_std_tests-c9e698994923ea6b.269mpyibt1n9z2i8.rcgu.o
            "_memcpy", referenced from:
                core::ops::function::FnMut::call_mut::h50a1ff7c808fa955 in subxt_core_no_std_tests-c9e698994923ea6b.1mdjt5lke7rk419d.rcgu.o
                frame_metadata::_::_$LT$impl$u20$parity_scale_codec..codec..Decode$u20$for$u20$frame_metadata..RuntimeMetadataPrefixed$GT$::decode::hc2d6160521da3f99 in subxt_core_no_std_tests-c9e698994923ea6b.1v8ak6ohszt9il8x.rcgu.o
                frame_metadata::_::_$LT$impl$u20$parity_scale_codec..codec..Decode$u20$for$u20$frame_metadata..RuntimeMetadata$GT$::decode::_$u7b$$u7b$closure$u7d$$u7d$::h02992c01d903eeee in subxt_core_no_std_tests-c9e698994923ea6b.1v8ak6ohszt9il8x.rcgu.o
                frame_metadata::_::_$LT$impl$u20$parity_scale_codec..codec..Decode$u20$for$u20$frame_metadata..RuntimeMetadata$GT$::decode::_$u7b$$u7b$closure$u7d$$u7d$::h0f8ac221182a9398 in subxt_core_no_std_tests-c9e698994923ea6b.1v8ak6ohszt9il8x.rcgu.o
                frame_metadata::_::_$LT$impl$u20$parity_scale_codec..codec..Decode$u20$for$u20$frame_metadata..RuntimeMetadata$GT$::decode::_$u7b$$u7b$closure$u7d$$u7d$::h36e492884daaee58 in subxt_core_no_std_tests-c9e698994923ea6b.1v8ak6ohszt9il8x.rcgu.o
                frame_metadata::_::_$LT$impl$u20$parity_scale_codec..codec..Decode$u20$for$u20$frame_metadata..RuntimeMetadata$GT$::decode::_$u7b$$u7b$closure$u7d$$u7d$::h3dc4f7f50ba2c6b4 in subxt_core_no_std_tests-c9e698994923ea6b.1v8ak6ohszt9il8x.rcgu.o
                frame_metadata::_::_$LT$impl$u20$parity_scale_codec..codec..Decode$u20$for$u20$frame_metadata..RuntimeMetadata$GT$::decode::_$u7b$$u7b$closure$u7d$$u7d$::h3fb7d05b52fb5ce6 in subxt_core_no_std_tests-c9e698994923ea6b.1v8ak6ohszt9il8x.rcgu.o
                ...
            "_abort", referenced from:
                _rust_begin_unwind in subxt_core_no_std_tests-c9e698994923ea6b.3g07tcecbxwnnq15.rcgu.o
            "_free", referenced from:
                _$LT$libc_alloc..LibcAlloc$u20$as$u20$core..alloc..global..GlobalAlloc$GT$::dealloc::hf2148cf9f27498b8 in subxt_core_no_std_tests-c9e698994923ea6b.269mpyibt1n9z2i8.rcgu.o
            "_bzero", referenced from:
                _$LT$libc_alloc..LibcAlloc$u20$as$u20$core..alloc..global..GlobalAlloc$GT$::alloc_zeroed::ha8abe505d0f3cbe3 in subxt_core_no_std_tests-c9e698994923ea6b.269mpyibt1n9z2i8.rcgu.o
                alloc::alloc::Global::grow_impl::h89bdf56ff6b1f1a7 in libparity_scale_codec-39751d065c7b75ba.rlib(parity_scale_codec-39751d065c7b75ba.parity_scale_codec.c0eb5f39e1cd9682-cgu.0.rcgu.o)
                alloc::alloc::Global::grow_impl::hc07514a638cb7bc8 in libtwox_hash-857443a2d443ea8b.rlib(twox_hash-857443a2d443ea8b.twox_hash.599fc9e923e5a001-cgu.0.rcgu.o)
          ld: symbol(s) not found for architecture arm64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: could not compile `subxt-core-no-std-tests` (bin "subxt-core-no-std-tests") due to 1 previous error
```